guides/bulk.md

```diff
@@ -0,0 +1,178 @@
+    { "create": { "data": { "title": "Beauty and the Beast 3", "year": "2031" } } },
+    { "create": { "data": { "title": "Beauty and the Beast 4", "year": "2049" } } },
+    { "create": { "_index": "books", "data": { "title": "The Lion King 2", "year": "1998" } } }
+  ]
+)
+```
+
+Note that we specified only the `_index` for the last document in the request body. This is because the `bulk` method accepts an `index` parameter that specifies the default `_index` for all bulk operations in the request body. Moreover, we omit the `_id` for each document and let OpenSearch generate them for us in this example, just like we can with the `create` method.
+
+### Updating multiple documents
+
+```python
+client.bulk(
+  "index": "movies",
+  [
+    { "update": { "_id": "1", "data": { "doc": { "year": "1992" } } } },
+    { "update": { "_id": "2", "data": { "doc": { "year": "2018" } } } }
+  ]
+)
+```
+
+Note that the updated data is specified in the `doc` field of the `data` object.
+
+### Deleting multiple documents
+
+```python
+client.bulk(
+  "index": "movies",
+  [
+    { "delete": { "_id": "1" } },
+    { "delete": { "_id": "2" } }
+  ]
+)
+```
+
+### Mix and match operations
+
+You can mix and match the different operations in a single request. The following code creates two documents, updates one document, and deletes another document:
+
+```python
+client.bulk(
+  "index": "movies",
+  [
+    { "create": { "data": { "title": "Beauty and the Beast 5", "year": "2050" } } },
+    { "create": { "data": { "title": "Beauty and the Beast 6", "year": "2051" } } },
+    { "update": { "_id": "3", "data": { "doc": { "year": "2052" } } } },
+    { "delete": { "_id": "4" } }
+  ]
+)
+```
+
+### Handling errors
+
+The `bulk` API returns an array of responses for each operation in the request body. Each response contains a `status` field that indicates whether the operation was successful or not. If the operation was successful, the `status` field is set to a `2xx` code. Otherwise, the response contains an error message in the `error` field.
+
+The following code shows how to look for errors in the response:
+
+```python
+response = client.bulk(
+  "index": "movies",
+  [
+    { "create": { "_id": "1", "data": { "title": "Beauty and the Beast", "year": "1991" } } },
+    { "create": { "_id": "2", "data": { "title": "Beauty and the Beast 2", "year": "2030" } } },
+    { "create": { "_id": "1", "data": { "title": "Beauty and the Beast 3", "year": "2031" } } }, # document already exists error
+    { "create": { "_id": "2", "data": { "title": "Beauty and the Beast 4", "year": "2049" } } }  # document already exists error
+  ]
+)
+
+for item in response["items"]
+  if 200 <= item.dig("create", "status") <= 299:
+    continue
+  print(item.dig('create', 'error', 'reason'))
+```
+
+## Cleanup
+
+To clean up the resources created in this guide, delete the `movies` and `books` indices:
+
+```python
+client.indices.delete("index": ["movies", "books"])
```

Comment:
@anabeel011, correct me if I am wrong. Above code didn't work for me. But instead this worked
`client.indices.delete(index= ["movies", "books"])`